src/main/java/talker/task/Task.java

```diff
@@ -0,0 +1,119 @@
+package talker.task;
+
+import talker.TalkerException;
+
+/**
+ * Represents a Task that has a description and can be marked as complete/incomplete
+ */
+public class Task {
+    // String describing task
+    protected String description;
+    // Status of task
+    protected boolean complete;
```

Comment:
Boolean field could be better named to reflect its boolean type.
```suggestion
    protected boolean isComplete;
```